### ISSUE TITLE
AMAZING PROGRESS!!

### ISSUE THREAD

**Issue by gratefulsorceror** [1kc/razer-macos#158]
Good stuff fellas. Awesome to see the project coming along so well. I already downloaded the newest version and I got some feedback; so using the wave setting at anything below normal produces very choppy visuals on the Black Widow Elite keyboard, the slowest slower and slow settings make the effect seem very rough and almost " bit by bit " is the best way I can describe it, not very fluent. I put it on faster setting and it looks much, much better; though I'm not exactly sure what the previous setting was for the wave, in any case I hope everyone has a happy New Years, take care y'all!

**Comment by akaler** [comment]
I hate bit by bit as much as you do. But I couldn't figure out how to make it smoother. For now just increase the speed to make it smoother. Some users have complained that slow isn't slow enough as well as we might be changing the settings to be even slower.

**Comment by macintoshm** [comment]
> I hate bit by bit rough animation as much as you do. But I couldn't figure out how to make it smoother. For now just increase the speed to make it smoother. Some users have complained that slow isn't slow enough. so I might add a turtle speed mode in there

I agree here, I think it could be even slower. Also thank you for this feature!
